arch/arm64/configs/enchilada_defconfig

```diff
@@ -742,18 +742,67 @@ CONFIG_AUTOFS4_FS=y
 
 CONFIG_LOCALVERSION="-hybris"
 
-# Recommended
+# Memory
 CONFIG_MEMCG=y
 CONFIG_MEMCG_KMEM=y
 CONFIG_MEMCG_SWAP=y
 
-
-# telnet
-
+# Telnet
 CONFIG_USB_F_RNDIS=y
 CONFIG_USB_CONFIGFS_RNDIS=y
 CONFIG_USB_F_GSI=n
 CONFIG_USB_CONFIGFS_F_GSI=n
 CONFIG_USB_CONFIGFS_F_QCGSI=n
 CONFIG_USB_CONFIGFS_QCRNDIS=n
-CONFIG_USB_CONFIGFS_RMNET_BAM=n 
\ No newline at end of file
+CONFIG_USB_CONFIGFS_RMNET_BAM=n
+
+# EXT4
+CONFIG_LBDAF=y # suport large ext4
+
+# CIFS
+CONFIG_CIFS=y
+CONFIG_BLK_DEV_NBD=y
+
+# NFS File System
+CONFIG_LOCKD=y
+CONFIG_LOCKD_V4=y
+CONFIG_NFS_V3_ACL=y
+CONFIG_NFS_FS=y
+CONFIG_NFS_COMMON=y
+CONFIG_NFS_V4=y
+CONFIG_NFS_V4_1=y
+CONFIG_NFS_V3=y
+CONFIG_SUNRPC_GSS=y
+CONFIG_NFS_USE_KERNEL_DNS=y
+CONFIG_SUNRPC=y
+CONFIG_NFS_ACL_SUPPORT=y
+
+# CD File Ssytem / Portable USB (OTG)
```

Comment:
I would keep this to personal builds for now, no need to clutter up the kernel at this point. It's also **extremely** unlikely that anyone will want these features.